frameworks/CSharp/aspnet/benchmark_config.json

```diff
@@ -203,7 +203,7 @@
       "notes": "",
       "versus": ""
     },
-    "mono": {
+    "default": {
```

Comment:
Why? You should be running `toolset/run-tests.py --mode verify --test aspnet-mono`, for example.